<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>740</width>
    <height>713</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
        <horstretch>1</horstretch>
        <verstretch>1</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>620</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="PX_TabWidget_main" name="tabWidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>1</horstretch>
         <verstretch>1</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>240</width>
         <height>0</height>
        </size>
       </property>
       <property name="sizeIncrement">
        <size>
         <width>1</width>
         <height>1</height>
        </size>
       </property>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>Tab 1</string>
        </attribute>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>Tab 2</string>
        </attribute>
       </widget>
      </widget>
      <widget class="QWidget" name="horizontalLayoutWidget">
       <layout class="QHBoxLayout" name="horizontalLayout"/>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>740</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuProgramm">
    <property name="title">
     <string>Program</string>
    </property>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menuProject">
    <property name="title">
     <string>Project</string>
    </property>
    <addaction name="actionNewProject"/>
    <addaction name="actionLoadProject"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
   </widget>
   <addaction name="menuProgramm"/>
   <addaction name="menuProject"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNewElement"/>
   <addaction name="actionNewPlus"/>
   <addaction name="actionNewMinus"/>
   <addaction name="actionNewMultiplication"/>
   <addaction name="actionNewDivision"/>
   <addaction name="separator"/>
   <addaction name="actionActivate_Simulation_Mode"/>
   <addaction name="actionRun"/>
   <addaction name="actionStop"/>
  </widget>
  <action name="actionClose">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="toolTip">
    <string>PxLinX beenden</string>
   </property>
  </action>
  <action name="actionNewElement">
   <property name="icon">
    <iconset>
     <normaloff>../Icons/element32.png</normaloff>../Icons/element32.png</iconset>
   </property>
   <property name="text">
    <string>neues Element</string>
   </property>
   <property name="toolTip">
    <string>neues Element einf√ºgen</string>
   </property>
  </action>
  <action name="actionNewConnector">
   <property name="icon">
    <iconset>
     <normaloff>../../../icons/conector32.png</normaloff>../../../icons/conector32.png</iconset>
   </property>
   <property name="text">
    <string>neuer Konnector</string>
   </property>
   <property name="toolTip">
    <string>neuer Konnector</string>
   </property>
  </action>
  <action name="actionNewPlus">
   <property name="icon">
    <iconset>
     <normaloff>../Icons/plus24.png</normaloff>../Icons/plus24.png</iconset>
   </property>
   <property name="text">
    <string>New Plus</string>
   </property>
  </action>
  <action name="actionNewMinus">
   <property name="icon">
    <iconset>
     <normaloff>../Icons/minus24.png</normaloff>../Icons/minus24.png</iconset>
   </property>
   <property name="text">
    <string>new Minus</string>
   </property>
  </action>
  <action name="actionNewDivision">
   <property name="icon">
    <iconset>
     <normaloff>../Icons/divide24.png</normaloff>../Icons/divide24.png</iconset>
   </property>
   <property name="text">
    <string>new Division</string>
   </property>
  </action>
  <action name="actionNewMultiplication">
   <property name="icon">
    <iconset>
     <normaloff>../Icons/multiply24.png</normaloff>../Icons/multiply24.png</iconset>
   </property>
   <property name="text">
    <string>New Multiplication</string>
   </property>
  </action>
  <action name="actionRun">
   <property name="icon">
    <iconset>
     <normaloff>../Icons/play24.png</normaloff>../Icons/play24.png</iconset>
   </property>
   <property name="text">
    <string>run</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save Project</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save Project As...</string>
   </property>
  </action>
  <action name="actionLoadProject">
   <property name="text">
    <string>Load Project</string>
   </property>
  </action>
  <action name="actionNewProject">
   <property name="text">
    <string>New Project</string>
   </property>
  </action>
  <action name="actionActivate_Simulation_Mode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../Icons/simulateMode24.png</normaloff>../Icons/simulateMode24.png</iconset>
   </property>
   <property name="text">
    <string>Activate Simulation Mode</string>
   </property>
  </action>
  <action name="actionMeasure">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../Icons/measure24.png</normaloff>../Icons/measure24.png</iconset>
   </property>
   <property name="text">
    <string>Measure</string>
   </property>
  </action>
  <action name="actionStimulate">
   <property name="icon">
    <iconset>
     <normaloff>../Icons/stimulate24.png</normaloff>../Icons/stimulate24.png</iconset>
   </property>
   <property name="text">
    <string>Stimulate</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="icon">
    <iconset>
     <normaloff>../Icons/stop24.png</normaloff>../Icons/stop24.png</iconset>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
   <property name="toolTip">
    <string>Stop Simulation</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PX_TabWidget_main</class>
   <extends>QTabWidget</extends>
   <header location="global">PyLinXGui.PX_TabWidget_main</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
